deploy-manage/monitor/access-performance-metrics-on-elastic-cloud.md

```diff
@@ -1,17 +1,17 @@
 ---
-mapped_pages:
+mapped_urls:
   - https://www.elastic.co/guide/en/cloud/current/ec-saas-metrics-accessing.html
   - https://www.elastic.co/guide/en/cloud-heroku/current/ech-saas-metrics-accessing.html
 applies_to:
   deployment:
     ess: all
 ---
 
-# Access performance metrics [ec-saas-metrics-accessing]
+# Performance metrics on {{ecloud}} [ec-saas-metrics-accessing]
```

Comment:
The analogue metrics for ECE would be the [platform collected metrics for all deployments](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/750/deploy-manage/monitor/orchestrators/ece-platform-monitoring), just in case you want to mention it.

I have this text [here](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/750/deploy-manage/deploy/cloud-enterprise/working-with-deployments) in case you find it useful:

> For Elasticsearch, Kibana, and Integrations Server components, use the External links to access each service's logs and metrics, including the associated proxy logs. These logs are part of [ECE platform monitoring], and are separate from user-configured stack monitoring.